surl/__init__.py

```diff
@@ -452,6 +456,14 @@ def main():
     # redirects, and --output - prints binary output to terminal
     arguments = ["curl", "-sSL", "--output", "-"]
 
+    for item in args.headers:
+        try:
+            k, v = [t.strip() for t in item.split(":")]
+        except ValueError:
+            print('Invalid header: "{}"'.format(item))
```

Comment:
I wouldn't bother mentioning it, but since I have one other comment on here already, I will:

Is it better to print this to `file=sys.stderr` ?

Reply:
Conceptually, that would make sense, but the original version of `surl` just prints to stdout (actually, this is copy+pasted from the old code) so I'm not going to change it just yet. Will definitely consider the change for a future 2.0 though.